.github/CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 [**Upgrade Guide**](https://intelowl.readthedocs.io/en/latest/Installation.html#update-to-the-most-recent-version)
 
+## [v3.0.0](https://github.com/intelowlproject/IntelOwl/releases/tag/v3.0.0)
+- **TLP Support**: Standardized threat-sharing using Traffic Light Protocol or `TLP`, thereby deprecating the use of booleans `force_privacy`, `disable_external_analyzers` and `private`. See [TLP Support](https://intelowl.readthedocs.io/en/master/Usage.html#tlp-support).
```

Comment:
Formatting nitpick:

Don't format the bold headings in a list. No need of `-` I mean.